Block/BlockContextManagerInterface.php

```diff
@@ -39,6 +39,13 @@ public function addSettingsByType($type, array $settings, $replace = false);
     public function addSettingsByClass($class, array $settings, $replace = false);
 
     /**
+     * @param string $type    Block type to check for
+     *
+     * @return bool
+     */
+    public function exists($type);
```

Comment:
IMHO it's not a good idea to remove the method from the interface. Yes, it is a BC break, but given that the method is called from within the bundle itself (from the Twig Extension), it should be marked as required in all implementations. But that's just my opinion.